src/pages/guides/dev-onboarding-how-is-it-unique/index.module.scss

```diff
@@ -1,13 +0,0 @@
-@import '@styles/colors';
-
-.hero {
-    background-image: url('/guides/dev-onboarding-unique/hero-mobile.png');
```

Comment:
Same thing here for these assets

Reply:
Same as above ^ 11b7fd7. Thank you for highlighting these!